Services/DataBaseService/Repositories/UserRepository.cs

```diff
@@ -67,5 +67,28 @@ public void DeleteUser(Guid userId)
             dbUserCredential.IsActive = false;
             dbContext.SaveChanges();
         }
+
+        public void EditUser(User user, PasswordHashChangeRequest password)
+        {
+            var dbUser = dbContext.Users.FirstOrDefault(uc => uc.Id == user.Id);
+            if (dbUser == null)
+            {
+                dbUser.LastName = user.LastName;
+                dbUser.FirstName = user.FirstName;
+                dbUser.Birthday = user.Birthday;
+                dbContext.SaveChanges();
+
+                if (password != null)
+                { 
+                    var dbUserCredential = dbContext.UsersCredentials.FirstOrDefault(uc => uc.UserId == user.Id);
+
+                    if (dbUserCredential != null)   dbUserCredential.PasswordHash = password.NewPasswordHash;

```

Comment:
1) `if` style guide?
2) Shouldn't we `SaveChanges()`?

Reply:
2.fix

Reply:
1 fix